test/kubemark/start-kubemark.sh

```diff
@@ -23,10 +23,15 @@ set -o pipefail
 TMP_ROOT="$(dirname "${BASH_SOURCE}")/../.."
 KUBE_ROOT=$(readlink -e ${TMP_ROOT} 2> /dev/null || perl -MCwd -e 'print Cwd::abs_path shift' ${TMP_ROOT})
 
-source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
 source "${KUBE_ROOT}/test/kubemark/cloud-provider-config.sh"
 source "${KUBE_ROOT}/test/kubemark/${CLOUD_PROVIDER}/util.sh"
 source "${KUBE_ROOT}/cluster/kubemark/${CLOUD_PROVIDER}/config-default.sh"
+
+if ls ${CLOUD_PROVIDER} | grep -Fq 'startup'; then
```

Comment:
This looks a bit odd. Could you clarify what you're intending to do?

Reply:
That line checks the cloud provider folder to see if a custom startup script exists, and then runs it if it does.